docs/docs/faq.md

```diff
@@ -12,7 +12,7 @@ and so which way depends on your use-case.
 
 The longer answer:
 
-- For global environments, you can specify the environment in `nebari_config.yml`, and it will be made available for all users and services (for example, CDSDashboards).
+- For global environments, you can specify the environment in `nebari_config.yml`, and it will be made available for all users and services.
```

Comment:
```suggestion
- For global environments, you can specify the environment in `nebari-config.yml`, and it will be made available for all users and services.
```